setup.cfg

```diff
@@ -27,6 +27,7 @@ zip_safe = False
 install_requires =
     protobuf >= 4.22
     grpcio >= 1.37.0
+    grpcio-status>=1.60.0
```

Comment:
```suggestion
    grpcio-status>=1.37.0
```

Reply:
It's important we use a low version here. Otherwise this forces all other users of the Dapr Python SDK to use this version with other tools that require it. And worse, grpcio-status 1.60.0 requires grpcio 1.60.0. And we cannot be totally sure that this is ok for every user of the Dapr Python SDK.